configman/config_manager.py

```diff
@@ -556,40 +556,45 @@ def get_option_names(self):
                 if isinstance(self.option_definitions[x], Option)]
 
     #--------------------------------------------------------------------------
-    def _create_reference_value_from_links(self, keys, known_keys):
+    def _create_reference_value_options(self, keys, finished_keys):
         """this method steps through the option definitions looking for
         alt paths.  On finding one, it creates the 'reference_value_from' links
         within the option definitions and populates it with copied options."""
         # a set of known reference_value_from_links
-        set_of_reference_value_from_links = set()
-        for key in (k for k in keys if k not in known_keys):
+        set_of_reference_value_option_names = set()
```

Comment:
by identifying the object with the prefix "set_of_...", the intent of the code is much clearer.  The type set implies to the code reader that this object is created over time, there are no duplicates and that the object will be used in set operations.  Without the prefix, the collection type and what it can do is unclear. To work with that object, one has to trace back through to code to determine the type.  

Prior to adoption of PEP8 standards, I used very long descriptive name for variables.  For me, this made it much simpler to understand code when revisiting it later.  I edited my long descriptive names so that they would fit the line length standards of PEP8.  To my eyes, code readability has suffered badly because of that.  I'm going to return to old practices as I believe readability is more important that the enforced brevity.